pkg/analysis/passes/sdkusage/sdkusage_test.go

```diff
@@ -76,6 +77,29 @@ func TestGoModNotParseable(t *testing.T) {
 }
 
 func TestValidGoMod(t *testing.T) {
+	httpmock.Activate()
+	defer httpmock.DeactivateAndReset()
+
+	// mock latest request
+	httpmock.RegisterResponder(
+		"GET",
+		"https://api.github.com/repos/grafana/grafana-plugin-sdk-go/releases/latest",
+		httpmock.NewStringResponder(
```

Comment:
I'm not used to this `httpmock` package but it seems that you have a `httpmock.NewJsonResponder` which seems a better fit? It sets the proper headers.

Reply:
For the record, we usually use the `httptest` package for this kind of tests, which is an upstream package and gives you more flexibility (it's also more low level though).

Reply:
Good idea about using the jsonresponser but I think it requires me to pass it an actual struct and not a json string which (IMO) is closer to what the network actually returns. I'lldo the change though, it makes it easier to read.

Regarding using `httptest` I tried using it but the amount of boilerplate and setup was not worth it for this simple test setup. Should we need some of those lower level APIs I'm sure we would use it. We also already use the httpmock package in other tests (of course introduced by me)